content/documentation/4-batch-developer-guides/2-batch/1-simple-task.md

```diff
@@ -1,41 +1,148 @@
+![BILL_STATMENTS](images/bill_statements.png)
 
 ### Initialzr
 
-Go to initializr - maybe have a curl command ready to go do we can just download a ready made .zip.
+1. Visit the [Spring Initialzr site](https://start.spring.io/).
+1. Select the latest release of spring boot.
+1. Create a new Maven project with a Group name of `io.spring` and an Artifact name of `billsetuptask`.
+1. In the Dependencies text box, type `task` to select the Cloud Task dependency.
+1. In the Dependencies text box, type `jdbc` then select the JDBC dependency.
+1. In the Dependencies text box, type `h2` then select the H2 dependency.
+   1. We use H2 for unit testing.
+1. In the Dependencies text box, type `mysql` then select mysql dependency(or your favorite database).
+   1. We use MySql for the runtime database.
+1. Click the Generate Project button.
+1. Unzip the billsetuptask.zip file and import the project into your favorite IDE.
+
+Another option instead of using the UI to initialize your project you can do the following:
+
+1. Execute the following curl command:
+
+   ```bash
+   curl https://start.spring.io/starter.zip -d language=java -d type=maven-project -d baseDir=billrun -d dependencies="batch,mysql,jdbc,h2,cloud-task" >billsetuptask.zip
+   ```
+
+2. Unzip the billsetuptask.zip file and import the project into your favorite IDE
+
+### Setting up MySql
+
+1. If you don't have an instance of MySql installed available to you, you can follow these instructions to run a MySql docker image for this example.
+
+   1. Pull the MySql docker image
+
+      ```bash
+      $ docker pull mysql:5.7.25
+      ```
+
+   2. Start the MySql
+
+      ```bash
+      $ docker run -p 3306:3306 --name some-mysql -e MYSQL_ROOT_PASSWORD=password  -e MYSQL_DATABASE=practice -d mysql:5.7.24
+      ```
 
 ### Biz Logic
 
-Your biz logic is coded in this section,
+Now let’s create the elements required for this application.
+
+1.  Using your IDE create the package `io.spring.billsetuptask.configuration`.
+1.  Create a [TaskConfiguration](https://github.com/cppwfs/edutasksamples/blob/master/billsetuptask/src/main/java/io/spring/billsetuptask/configuration/TaskConfiguration.java) class in the `io.spring.billsetuptask.configuration` package using your favorite IDE that looks like the contents below.
+
+```java
+{/* highlight-range{2} */}
+@Configuration
+@EnableTask
+public class TaskConfiguration {
+
+    @Autowired
+    private DataSource dataSource;
+
+    @Bean
+    public CommandLineRunner commandLineRunner() {
+        return args -> {
+            JdbcTemplate jdbcTemplate = new JdbcTemplate(dataSource);
+            jdbcTemplate.execute("CREATE TABLE IF NOT EXISTS " +
+                    "BILL_STATEMENTS ( id int, " +
+                    "first_name varchar(50), last_name varchar(50), " +
+                    "minutes int,data_usage int, bill_amount double)");
+        };
+    }
+}
+```
```

Comment:
I think this setup task needs to do more. I'm not sure I would ever write a boot app just to do this. Maybe setup some audit tables, and if the table exists, truncate it, record the event.

Reply:
We can chat with Michael and see if we can dream up something a little more interesting.